### DISCUSSION TITLE
Dúvida

### DISCUSSION THREAD

**Discussion by daniel-canesso** [apyb/comunidade#118] [Geral]
Pessoal, bom dia!

Estou precisando de uma carta da associação, informando que eu sou membro. A quem devo solicitar, ou entrar em contato?

**Comment by anicelysantos** [comment]
Oi, Daniel.

Envia a solicitação, por favor, para diretoria@python.org.br.

**Reply by daniel-canesso** [reply]
Anicely, bom dia!
Obrigado, acabei de enviar.
